.github/workflows/github_release_automation.yml

```diff
@@ -0,0 +1,20 @@
+name: Release
+
+on:
+  push:
+    branches:
+    - master
+
+jobs:
+  build:
+    name: Release
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+    - id: create_release
+      name: Create Release
```

Comment:
This step does not create releases actually, but updates the draft. Could you update the step name to avoid confusion?

Reply:
Oh, yes sry